internal/util/log/log.go

```diff
@@ -154,3 +159,38 @@ func TraceLog(ctx context.Context, message string, args ...interface{}) {
 		klog.InfoDepth(1, logMessage)
 	}
 }
+
+// GzipLogFile convert log file from text format gzip compressed format.
+func GzipLogFile(pathToFile string) error {
```

Comment:
Consider placing this in a new file in the same directory.

You also should add a unit-test for new helper functions (in the same commit). Demonstrate that a new file was created and is in gzip format. I wonder if the original file should get removed/truncated?

Reply:
Yes! The original file will get removed to save the space with the `compress` strategy.